Mathlib/FieldTheory/Finite/Basic.lean

```diff
@@ -179,6 +179,33 @@ theorem forall_pow_eq_one_iff (i : ℕ) : (∀ x : Kˣ, x ^ i = 1) ↔ q - 1 ∣
       rw [← pow_mul, mul_comm, pow_mul, h, one_pow]
 #align finite_field.forall_pow_eq_one_iff FiniteField.forall_pow_eq_one_iff
 
+theorem cast_subgroup_of_units_card_ne_zero {F : Type} [Field F]
```

Comment:
```suggestion
theorem card_cast_subgroup_card_ne_zero {F : Type} [Field F]
```

maybe? I feel like it must be a subgroup of the units automatically anyways, I don't like this name 100% either but better than the original name (the `_card` gets lost within the `_of_units`)